api.planx.uk/modules/send/bops/bops.ts

```diff
@@ -44,11 +44,13 @@ const sendToBOPS = async (req: Request, res: Response, next: NextFunction) => {
   // a local or staging API instance should send to the BOPS staging endpoint
   // production should send to the BOPS production endpoint
   const localAuthority = req.params.localAuthority;
-  const env = process.env.NODE_ENV === "production" ? "production" : "staging";
+  const env =
+    process.env.APP_ENVIRONMENT === "production" ? "production" : "staging";
```

Comment:
I'd made the assumption that `NODE_ENV` would be a reliable indicator here, but of course we use a production build for E2E tests.